src/frontend/test_runner/tests/testdata/tpch.yaml

```diff
@@ -1008,6 +1008,42 @@
                       BatchProject { exprs: [$0] }
                         BatchFilter { predicate: ($1 = 'ARGENTINA':Varchar) }
                           BatchScan { table: nation, columns: [n_nationkey, n_name] }
+  stream_plan: |
+    StreamMaterialize { columns: [ps_partkey, value], pk_columns: [ps_partkey], order_descs: [value, ps_partkey] }
+      StreamExchange { dist: HashShard([0]) }
+        StreamProject { exprs: [$0, $2] }
+          StreamDynamicFilter { predicate: ($3 > $4) }
+            StreamExchange { dist: HashShard([3]) }
+              StreamHashAgg { group_keys: [$0], aggs: [count, sum($1), sum($1)] }
+                StreamExchange { dist: HashShard([0]) }
+                  StreamProject { exprs: [$0, ($2 * $1), $3, $4, $5] }
+                    StreamHashJoin { type: Inner, predicate: $3 = $6, output_indices: [0, 1, 2, 4, 5, 7] }
+                      StreamExchange { dist: HashShard([3]) }
+                        StreamHashJoin { type: Inner, predicate: $1 = $5, output_indices: [0, 2, 3, 6, 4, 7] }
+                          StreamExchange { dist: HashShard([1]) }
+                            StreamTableScan { table: partsupp, columns: [ps_partkey, ps_suppkey, ps_availqty, ps_supplycost, _row_id], pk_indices: [4] }
+                          StreamExchange { dist: HashShard([0]) }
+                            StreamTableScan { table: supplier, columns: [s_suppkey, s_nationkey, _row_id], pk_indices: [2] }
+                      StreamExchange { dist: HashShard([0]) }
+                        StreamProject { exprs: [$0, $1] }
+                          StreamFilter { predicate: ($2 = 'ARGENTINA':Varchar) }
+                            StreamTableScan { table: nation, columns: [n_nationkey, _row_id, n_name], pk_indices: [1] }
+            StreamProject { exprs: [($1 * 0.0001000000:Decimal)] }
+              StreamExchange { dist: Broadcast }
```

Comment:
For some weird reason, `Exchange` is the chiild of `Project`. Trying to figure out why...

Reply:
It's weird, this scenario only occurs for `Broadcast` but not, for instance `SomeShard`...

Reply:
Any ideas? @st1page

Reply:
Ok, fixed, turns out it was not matching on Broadcast somewhere...